### ISSUE TITLE
Account nuked?

### ISSUE THREAD

**Issue by Sebbito** [dessalines/jerboa#633]
** Jerboa Version **
0.0.33

**Describe the bug**
My main instance was lemmy.ml but right now it seems like my account got nuked.
My account settings are completely empty, my feed is empty and i can't add a new account with my credentials under another instance (tho I'm not sure if that's how that works but i assumed that accounts worked cross instance).

**To Reproduce**
Steps to reproduce the behaviour:

1. Open App
2. See empty feed
3. Open account settings
4. See empty settings

**Comment by MV-GH** [comment]
Is it similar to this bug ? https://github.com/dessalines/jerboa/issues/547

**Comment by Sebbito** [comment]
Yes the symptoms are the same.

I'm not sure if an update caused the issues or not. I let the Play Store auto-update my apps so idk if that happened after any update took place or not.

Also i just opened the app and it's back to normal.

What i think happened:
The instance lemmy.ml seems to have been down during the time that the issue persisted (i can now reach it again). I got a 502 when trying to reach lemmy.ml with Firefox before.
That is also the instance on which I created the account and is set in the app.
I assume that the app also got a 502 and couldn't update correctly because of it.

I think the only thing that can be done is notify the user that the site is currently not available.
